Formula/unzip.rb

```diff
@@ -72,7 +77,12 @@ def install
     (testpath/"test2").write "Bonjour!"
     (testpath/"test3").write "Hej!"
 
-    system "/usr/bin/zip", "test.zip", "test1", "test2", "test3"
```

Comment:
Can't it just use `zip` from PATH?

Reply:
Yes, we can do that for Mac. But on Linux, zip is keg-only, and some systems do not provide zip (like our CI for instance).

If we want to simplify it, we could use our zip formula for the Mac and the Linux test.

Reply:
Ah, forgot about keg_only